packages/atomic/src/components/atomic-result/atomic-result.pcss

```diff
@@ -1,440 +1,21 @@
-
-    atomic-result-section-visual {
-      width: 10rem;
-      height: 10rem;
-
-      @screen mobile {
-        margin-right: 1rem;
-        margin-bottom: 1rem;
-        width: 24vw;
-        height: 24vw;
-      }
-    }
-  }
-
-  &.image-icon {
-    grid-template-areas:
-      'badges badges          .               actions'
-      'visual title           title           title'
-      'visual title-metadata  title-metadata  title-metadata'
-      'visual emphasized      emphasized      emphasized'
-      'visual excerpt         excerpt         excerpt'
-      'visual bottom-metadata bottom-metadata bottom-metadata'
-      'visual .               .               .';
-    grid-template-columns: minmax(0, min-content) auto 1fr auto;
-    grid-template-rows: repeat(6, auto) 1fr;
-
-    @screen mobile {
-      grid-template-areas:
-        'badges          badges'
-        'visual          title'
-        'visual          title-metadata'
-        'visual          emphasized'
-        'visual          excerpt'
-        'visual          bottom-metadata'
-        'visual          actions'
-        'visual          .';
-      grid-template-columns: auto minmax(0, 1fr);
-      grid-template-rows: repeat(7, auto) 1fr;
-    }
-
-    atomic-result-section-visual {
-      width: 2rem;
-      height: 2rem;
-
-      @screen mobile {
-        margin-right: 1rem;
-        margin-bottom: 0;
-      }
-    }
-  }
-
-  &.image-none {
-    grid-template-areas:
-      'badges          .               actions'
-      'title           title           title'
-      'title-metadata  title-metadata  title-metadata'
-      'emphasized      emphasized      emphasized'
-      'excerpt         excerpt         excerpt'
-      'bottom-metadata bottom-metadata bottom-metadata';
-    grid-template-columns: auto 1fr auto;
-    grid-template-rows: repeat(6, auto);
-
-    @screen mobile {
-      grid-template-areas:
-        'badges'
-        'title'
-        'title-metadata'
-        'emphasized'
-        'excerpt'
-        'bottom-metadata'
-        'actions';
-      grid-template-columns: 100%;
-      grid-template-rows: repeat(7, auto);
-    }
-
-    atomic-result-section-visual {
-      display: none;
-    }
-  }
-
-  &.image-small,
-  &.image-icon,
-  &.image-none {
-    @screen mobile {
-      border: 1px solid var(--atomic-neutral);
-      margin: 1rem;
-      padding: 1rem;
-      border-radius: 1rem;
-    }
-  }
-}
-
-/* == Density styles == */
-.result-root.density-comfortable {
-  margin: 0 1rem;
-  padding: 2rem 0;
-
-  atomic-result-section-visual {
-    margin-right: 2rem;
-  }
+  @apply common-result;
 
-  atomic-result-section-badges {
-    margin-bottom: 1.5rem;
-    height: 2.5em;
-
-    @screen mobile {
-      height: 2rem;
-    }
-  }
-
-  atomic-result-section-actions {
-    margin-bottom: 1.5rem;
-    height: 2.5em;
-
-    @screen mobile {
-      margin-top: 1rem;
-      margin-bottom: 0;
-      height: 2.5rem;
+  &.display-list {
+    @screen desktop {
```

Comment:
I didn't want the mobile look to override the desktop look nor the other way around. This made everything very confusing to read, since some densities and image sizes were also overriding other attributes.